### PR TITLE
Added Soham Sonawane in members

### LINKED ISSUE #1687: [Community] Member Profile: Soham Sonawane
**Current Behavior**

@killbotXD has been a consistent contributor and community member.
**Desired Situation**
Let's recognize @killbotXD as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @killbotXD 
- Twitter: @killbot_XD
- LinkedIn: soham-sonawane-66b606178
- Link to profile picture: [link](https://drive.google.com/uc?export=view&id=1kV-nlPMm9yB3P9LOL6obw35kAPurEZLI)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
**Description**

This PR fixes #1687 .

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/soham-sonawane/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Soham Sonawane
+position: Contributor
+image_path: ./soham-sonawane.jpg
+github: @killbotXD
```
Comment (Tanuj22) [inline src/collections/members/soham-sonawane/index.mdx:5]:
```suggestion
github: killbotXD
```

File: src/collections/members/soham-sonawane/index.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+twitter: @killbot_XD
```
Comment (Tanuj22) [inline src/collections/members/soham-sonawane/index.mdx:6]:
```suggestion
twitter: killbot_XD
```

File: src/collections/members/soham-sonawane/index.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+linkedin: soham-sonawane-66b606178
+location: Pune, India
+bio: An Otaku who is also a developer ;) , loves to design beautiful stuff, is also an awesome photographer and can 
+     debug 99.9% of the bugs out there.
+status: Active
+published: true
+---
+
```
Comment (Tanuj22) [inline src/collections/members/soham-sonawane/index.mdx:14]: Remove these empty lines

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: src/collections/members/soham-sonawane/index.mdx (hunk 3 above)
Comment (96RadhikaJadhav) [reply src/collections/members/soham-sonawane/index.mdx]: @Tanuj22  Done.

**Comment by l5io** [comment]
🚀 Preview for commit ee98f3ea59736d32cfa64403bd196245bf0116cd at: https://604a6f5c2a42ad23c817d322--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 36a6e5b490f3c317f265ebe338ff8072f209db3d at: https://604a76b153d5d7275c35b8ae--layer5ng.netlify.app

File: src/collections/members/soham-sonawane/index.mdx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+name: Soham Sonawane
+position: Contributor
+image_path: ./soham-sonawane.jpg
+github: killbotXD
+twitter: killbot_XD
+linkedin: soham-sonawane-66b606178
+location: Pune, India
+bio: An Otaku who is also a developer ;) , loves to design beautiful stuff, is also an awesome photographer and can 
+     debug 99.9% of the bugs out there.
```
Comment (killbotXD) [inline src/collections/members/soham-sonawane/index.mdx:10]: this newline wont cause an Issue right?

Comment (Tanuj22) [reply src/collections/members/soham-sonawane/index.mdx:10]: Nope

Comment (killbotXD) [reply src/collections/members/soham-sonawane/index.mdx:10]: cool lgtm

**Review by killbotXD — APPROVED** [review]

**Review by Tanuj22 — APPROVED** [review]
Thank you @96RadhikaJadhav :)

**Comment by leecalcote** [comment]
Yes, thank you, @96RadhikaJadhav! 😎

@killbotXD 🥳

**Comment by killbotXD** [comment]
thank you @96RadhikaJadhav  :) thanks @leecalcote  @dhruv0000 @Tanuj22 @everyone

**Pull request merged**